drivers/soundwire/master.c

```diff
@@ -7,6 +7,7 @@
 #include <linux/soundwire/sdw_type.h>
 #include "bus.h"
 
+/* nothing to free but this function is mandatory */
```

Comment:
@bardliao hmm you say this but you are freeing md?

Reply:
Fixed